File: Frontend-POS/RBMS-POS-Client/src/app/core/api/models/operating-hour-model.ts
/* tslint:disable */
/* eslint-disable */
import { EDayOfWeek } from '../models/e-day-of-week';
export interface OperatingHourModel {
  closeTime1?: string | null;
  closeTime2?: string | null;
  dayOfWeek?: EDayOfWeek;
  isOpen?: boolean;
  openTime1?: string | null;
  openTime2?: string | null;
  shopOperatingHourId?: number;
}

File: Backend-POS/POS.Main/POS.Main.Business.Admin/Models/ShopSettings/OperatingHourModel.cs
using POS.Main.Core.Enums;

namespace POS.Main.Business.Admin.Models.ShopSettings;

public class OperatingHourModel
{
    public int ShopOperatingHourId { get; set; }
    public EDayOfWeek DayOfWeek { get; set; }
    public bool IsOpen { get; set; }
    public string? OpenTime1 { get; set; }
    public string? CloseTime1 { get; set; }
    public string? OpenTime2 { get; set; }
    public string? CloseTime2 { get; set; }
}
